fleet_notifications/incoming_call_endpoint.py

```
@@ -0,0 +1,123 @@
+
+
+class FlaskAppWrapper(object):
+    def __init__(self, app, **configs):
+        self.app = app
+        self.configs(**configs)
+
+    def configs(self, **configs):
+        for config, value in configs:
+            self.app.config[config.upper()] = value
+
+    def add_endpoint(self, endpoint=None, endpoint_name=None, handler=None, methods=['GET'], *args, **kwargs):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)